packages/aws-cdk-lib/cloudformation-include/test/test-templates/non-existent-policy-attribute.json

```diff
@@ -1,17 +1,32 @@
 {
+  "AWSTemplateFormatVersion": "2010-09-09",
   "Resources": {
-    "ASG": {
+    "AutoScalingGroup": {
       "Type": "AWS::AutoScaling::AutoScalingGroup",
       "Properties": {
+        "DesiredCapacity": "1",
         "MinSize": "1",
         "MaxSize": "5"
       },
       "CreationPolicy": {
-        "NonExistentResourceAttribute": "Bucket1"
+        "ResourceSignal": {
+          "Count": 1,
+          "Timeout": "PT10M"
+        }
```

Comment:
Why are you renaming properties in a file called `non-existent-policy-attribute.json` away from a nonexistent name to an recognized name?